.github/workflows/tuist.yml

```diff
@@ -114,6 +113,44 @@ jobs:
           bundle install --jobs 4 --retry 3
       - name: Run tests
         run: FEATURE=features/${{ matrix.feature }}.feature bundle exec rake features
+  xcode12_specific_acceptance_tests: # Remove once all other acceptance tests support Xcode 12
```

Comment:
Nitpick: I'd rename this one to just `xcode_12_acceptance_tests`. Also notice that the `cache_acceptance_tests` have also been moved to a separate job for the same reason. I'd suggest merging both into the same.